cyto_dl/datamodules/czi.py

```diff
@@ -3,7 +3,7 @@
 
 import numpy as np
 import pandas as pd
-from aicsimageio.aics_image import AICSImage
+from bioio import BioImage
```

Comment:
The API for `BioImage` and `AICSImage` is, as far as I can tell, identical. So, all we really need to do is swap out `AICSImage` for `BioImage` wherever we used it.